contracts/varatube/src/utils.rs

```diff
@@ -0,0 +1,132 @@
+use fungible_token_io::FTEvent;
+
+use crate::{msg, Actions, ActorId, Error, Period, Price, SubscriberData, VaraTube};
+
+impl VaraTube {
+    /// Add subscriber.
+    pub fn add_subscriber(&mut self, subscriber: &ActorId, data: SubscriberData) {
+        self.subscribers.insert(*subscriber, data);
+    }
+
+    /// Add pending subscription.
+    ///
+    /// Inserting `data` is actually currency id and subscription period.
+    pub fn add_pending_subscriber(&mut self, subscriber: &ActorId, data: (ActorId, Period)) {
+        let (currency_id, period) = data;
```

Comment:
We can destruct params in arg list.
```suggestion
    pub fn add_pending_subscriber(&mut self, subscriber: &ActorId, (currency_id, period): (ActorId, Period)) {
```